src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicator.java

```diff
@@ -80,13 +104,12 @@
  * @since 1.0
  */
 public class AverPTZCommunicator extends UDPCommunicator implements Controller, Monitorable {
+	private final ReentrantLock controlOperationsLock = new ReentrantLock();
 	private int cameraID = 1;
 	private int panSpeed = 1;
 	private int tiltSpeed = 1;
 	private int sequenceNumber = 0;
-
 	private AverPTZRestCommunicator restCommunicator;
```

Comment:
In the AverPTZRestCommunicator class (not this class), I see you are using 
import static com.avispl.symphony.dal.communicator.aver.ptz.AverPTZConstants.*;

Please only import what you need instead of all of them